microcosm/tracing.py

```diff
@@ -0,0 +1,31 @@
+from jaeger_client import Config
+
+from microcosm.api import binding, defaults, typed
+
+
+TRACE_ID = "uber-trace-id"
+SPAN_NAME = "span_name"
+
+
+@binding("tracer")
+@defaults(
+    sample_type="ratelimiting",
+    sample_param=typed(int, 10),
+)
+def configure_tracing(graph):
+    """
+    See https://www.jaegertracing.io/docs/1.12/sampling/ for more info about
+    available sampling strategies.
+
+    """
+    config = Config(
+        config={
+            'sampler': {
```

Comment:
nit: double-quotes (maybe we should auto-lint on this).

Reply:
got it